app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx

```diff
@@ -14,6 +14,12 @@ const ButtonText = styled.Text<{ active: boolean }>(({ theme, active }) => ({
   fontSize: 11,
 }));
 
+const styles = StyleSheet.create({
+  focussedButton: {
+    backgroundColor: 'grey',
```

Comment:
I'm wondering if maybe there is some focus styling we can copy from the web storybook, changing the background is ok but probably there is some nice styling defined already for this. I think the web in some cases uses a blue outline.

Reply:
Will this style be ok?
<img width="158" alt="Screen Shot 2022-06-28 at 10 36 53" src="https://user-images.githubusercontent.com/83782787/176233672-ff78ac44-a43c-41dc-865c-b2874ca86aaa.png">

Reply:
got it from web storybook

Reply:
I think that should be fine, although I guess this is the selected style.

In this screenshot you can see the focused item has only the blue line and the text is grey

![image](https://user-images.githubusercontent.com/3481514/176241838-01c30142-6f1e-407a-be49-97d576fb6da3.png)